plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -117,8 +134,8 @@ def test_jobs_configured_via_config(sweep_runner: TSweepRunner) -> None:
         assert len(returns) == 2
         best_parameters = returns["ax"]
         assert len(best_parameters) == 2
-        assert math.isclose(best_parameters["quadratic.x"], 0.0, abs_tol=1e-4)
-        assert math.isclose(best_parameters["quadratic.y"], -1.0, abs_tol=1e-4)
+        assert math.isclose(best_parameters["quadratic_x"], 0.0, abs_tol=1e-4)
+        assert math.isclose(best_parameters["quadratic_y"], -1.0, abs_tol=1e-4)
```

Comment:
hmm, I don't like this, but I guess it's okay for now.
we can change the output format to:
```
best_parameters:
  - key: quadratic.x
    value: 0.11
  - key: quadratic.y
    value: 0.231
```

Testing would be more cumbersome but this would be speaking in the language of the end user.

Reply:
I dont mind changing it. Though the user uses both "quadratic.x" and "quadratic_x" depending on where they are specifying it.